src/game/area.c

```diff
@@ -85,13 +85,11 @@ Vp gViewport = { {
     { 640, 480, 511, 0 },
 } };
 
-#if MULTILANG
-const char *gNoControllerMsg[] = {
+langarray_t gNoControllerMsg = DEFINE_LANGUAGE_ARRAY(
```

Comment:
erm actually the `_t` suffix is reserved for builtin POSIX types 🤓

Reply:
what do you want me to change it to nerd